app/extensions/brave/locales/bn-BD/app.properties

```diff
@@ -0,0 +1,111 @@
+reload=Reload
```

Comment:
Yes, that is what Transfex does when an identifier is not present in the destination language